master/buildbot/test/fake/fakedb.py

```diff
@@ -528,6 +533,7 @@ class LogChunk(Row):
         compressed=0)
 
     required_columns = ('logid', )
+    utf8_columns = ('content', )
```

Comment:
FakeDB emulates models tables.
In real model's tables Binary fields are read and written only as bytestrings.
When unicode is passed from the outside into DB layer's Binary field writer it's being handled in the following way:
1) if driver is pg8000: value is encoded back by SQLAlchemy to bytestring, effectively as `str(value)` (which of course fails if `value` is not ascii)
2) if driver is psycopg2: passing unicode value into Binary column is an error.

IMO behaviour of psycopg2 is correct and it was luck that in pg8000 that code worked.
This PR fixes raised error when psycopg2 driver is used.